---
title: "2022 - The R package 'Anaconda' Targeted Differential and Global Enrichment Analysis of Taxonomic Rank by Shared Asvs"
collection: publications
permalink: /publication/Stenger_2022b
excerpt: 'Targeted differential and global enrichment analysis of taxonomic rank by shared ASVs (Ampli- con Sequence Variant), for high-throughput eDNA sequencing of fungi, bacteria, and metazoan. Actually works in two steps: I) Targeted differential analysis from QI- IME2 data and II) Global analysis by Taxon Mann-Whitney U test analysis from targeted analy- sis (I) (I) Estimate variance-mean dependence in count/abundance ASVs data from high-throughput se- quencing assays and test for differential represented ASVs based on a model using the nega- tive binomial distribution. (II) NCBITaxon_MWU uses continuous measure of significance (such as fold-change or -log(p- value)) to identify NCBITaxon that are significantly enriches with either up- or down- represented ASVs. If the measure is binary (0 or 1) the script will perform a typi- cal 'NCBITaxon enrichment' analysis based Fisher's exact test: it will show NCBITaxon over- represented among the ASVs that have 1 as their measure. On the plot, differ- ent fonts are used to indicate significance and color indicates enrichment with ei- ther up (red) or down (blue) regulated ASVs. No colors are shown for binary measure analy- sis. The tree on the plot is hierarchical clustering of NCBITaxon based on shared ASVs. Cate- gories with no branch length between them are subsets of each other. The fraction next to the cat- egory name indicates the fraction of 'good' ASVs in it; 'good' ASVs are the ones exceed- ing the arbitrary absValue cutoff (option in taxon_mwuPlot()). For Fisher's based test, specify ab- sValue=0.5. This value does not affect statistics and is used for plotting only. The origi- nal idea was for genes differential expression analy- sis from Wright et al (2015) <doi:10.1186/s12864-015-1540-2>; adapted here for taxo- nomic analysis. The 'Anaconda' package makes it possible to carry out these analyses by automatically creat- ing several graphs and tables and storing them in specially created subfold- ers. You will need your QIIME2 pipeline output for each kingdom (eg; Fungi and/or Bacte- ria and/or Metazoan): i) taxonomy.tsv, ii) taxonomy_RepSeq.tsv, iii) ASV.tsv and iv) Sam- pleSheet_comparison.txt (the latter being created by you).'
date: 2022-10-14
venue: 'CRAN'
paperurl: 'https://cran.r-project.org/web/packages/Anaconda/index.html'
citation: 'Stenger, P.-L. The R Package “Anaconda”: Targeted Differential and Global Enrichment Analysis of Taxonomic Rank by Shared Asvs <i>2022, CRAN 1–28. http://dx.doi.org/10.13140/RG.2.2.11117.67048.</i>'
---
Targeted differential and global enrichment analysis of taxonomic rank by shared ASVs (Amplicon Sequence Variant), for high-throughput eDNA sequencing of fungi, bacteria, and metazoan. Actually works in two steps: I) Targeted differential analysis from QI- IME2 data and II) Global analysis by Taxon Mann-Whitney U test analysis from targeted analy- sis (I) (I) Estimate variance-mean dependence in count/abundance ASVs data from high-throughput se- quencing assays and test for differential represented ASVs based on a model using the nega- tive binomial distribution. (II) NCBITaxon_MWU uses continuous measure of significance (such as fold-change or -log(p- value)) to identify NCBITaxon that are significantly enriches with either up- or down- represented ASVs. If the measure is binary (0 or 1) the script will perform a typi- cal 'NCBITaxon enrichment' analysis based Fisher's exact test: it will show NCBITaxon over- represented among the ASVs that have 1 as their measure. On the plot, differ- ent fonts are used to indicate significance and color indicates enrichment with ei- ther up (red) or down (blue) regulated ASVs. No colors are shown for binary measure analy- sis. The tree on the plot is hierarchical clustering of NCBITaxon based on shared ASVs. Cate- gories with no branch length between them are subsets of each other. The fraction next to the cat- egory name indicates the fraction of 'good' ASVs in it; 'good' ASVs are the ones exceed- ing the arbitrary absValue cutoff (option in taxon_mwuPlot()). For Fisher's based test, specify ab- sValue=0.5. This value does not affect statistics and is used for plotting only. The origi- nal idea was for genes differential expression analy- sis from Wright et al (2015) <doi:10.1186/s12864-015-1540-2>; adapted here for taxo- nomic analysis. The 'Anaconda' package makes it possible to carry out these analyses by automatically creat- ing several graphs and tables and storing them in specially created subfold- ers. You will need your QIIME2 pipeline output for each kingdom (eg; Fungi and/or Bacte- ria and/or Metazoan): i) taxonomy.tsv, ii) taxonomy_RepSeq.tsv, iii) ASV.tsv and iv) Sam- pleSheet_comparison.txt (the latter being created by you).
[Download paper here](https://cran.r-project.org/web/packages/Anaconda/index.html)

Recommended citation: 'Stenger, P.-L. The R Package “Anaconda”: Targeted Differential and Global Enrichment Analysis of Taxonomic Rank by Shared Asvs <i>2022, CRAN 1–28. http://dx.doi.org/10.13140/RG.2.2.11117.67048.</i>'
