---
title: "2022 - The R package Anaconda : Targeted differential and Global enrichment analysis of taxonomic rank by shared ASVs"
collection: publications
permalink: /publication/Stenger_2022c
excerpt: 'Targeted differential and global enrichment analysis of taxonomic rank by shared ASVs (Amplicon Sequence Variant), for high-throughput eDNA sequencing of fungi, bacteria, and metazoan. Actually works in two steps: I) Targeted differential analysis from QIIME2 data and II) Global analysis by Taxon Mann-Whitney U test analysis from targeted analysis (I) (I) Estimate variance-mean dependence in count/abundance ASVs data from high-throughput sequencing assays and test for differential represented ASVs based on a model using the negative binomial distribution. (II) NCBITaxon_ MWU uses continuous measure of significance (such as fold-change or -log(p- value)) to identify NCBITaxon that are significantly enriches with either up- or down- represented ASVs. If the measure is binary (0 or 1) the script will perform a typical 'NCBITaxon enrichment' analysis based Fisher's exact test: it will show NCBITaxon over- represented among the ASVs that have 1 as their measure. On the plot, differ- ent fonts are used to indicate significance and color indicates enrichment with either up (red) or down (blue) regulated ASVs. No colors are shown for binary measure analysis. The tree on the plot is hierarchical clustering of NCBITaxon based on shared ASVs. Categories with no branch length between them are subsets of each other. The fraction next to the category name indicates the fraction of 'good' ASVs in it; 'good' ASVs are the ones exceeding the arbitrary absValue cutoff (option in taxon_ mwuPlot()). For Fisher's based test, specify absValue=0.5. This value does not affect statistics and is used for plotting only. The original idea was for genes differential expression analysis from Wright et al (2015); adapted here for taxonomic analysis. The 'Anaconda' package makes it possible to carry out these analyses by automatically creat- ing several graphs and tables and storing them in specially created subfold- ers. You will need your QIIME2 pipeline output for each kingdom (eg; Fungi and/or Bacteria and/or Metazoan): i) taxonomy.tsv, ii) taxonomy_ RepSeq.tsv, iii) ASV.tsv and iv) SampleSheet_comparison.txt (the latter being created by you).'
date: 2021-11-09
venue: 'Evolutionary Applications'
paperurl: 'https://onlinelibrary.wiley.com/doi/abs/10.1111/eva.13464'
citation: 'Stenger PL., Ky CL., Vidal Dupiol J., Planes S., Reisser C. (2022) - Identifying genes associated with genetic control of color polymorphism in the pearl oyster Pinctada margaritifera var. cumingii (Linnaeus 1758) using a comparative whole genome pool-sequencing approach – <i>Evolutionary Applications, https://doi.org/10.1111/eva.13464</i>'
---
For hundreds of years, the color diversity of Mollusca shells has been a topic of interest for humanity. However, the genetic control underlying color expression is still poorly understood in mollusks. The pearl oyster *Pinctada margaritifera* is increasingly becoming a biological model to study this process due to its ability to produce a large range of colors. Previous breeding experiments demonstrated that color phenotypes were partly under genetic control, and while a few genes were found in comparative transcriptomics and epigenetic experiments, genetic variants associated with the phenotypes have not yet been investigated. Here, we used a pooled-sequencing approach on 172 individuals to investigate color-associated variants on three color phenotypes of economic interest for pearl farming, in three wild and one hatchery populations. While our results uncovered SNPs targeting pigment-related genes already identified in previous studies, such as PBGD, tyrosinases, GST, or FECH, we also identified new color-related genes occurring in the same pathways, like CYP4F8, CYP3A4, and CYP2R1. Moreover, we identified new genes involved in novel pathways unknown to be involved in shell coloration for *P. margaritifera*, like the carotenoid pathway, BCO1. These findings are essential to possibly implement future breeding programs focused on individual selection for specific color production in pearl oysters and improve the footprint of perliculture on the Polynesian lagoon by producing less but with a better quality.
[Download paper here](https://onlinelibrary.wiley.com/doi/abs/10.1111/eva.13464)

Recommended citation: 'Stenger PL., Ky CL., Vidal Dupiol J., Planes S., Reisser C. (2022) - Identifying genes associated with genetic control of color polymorphism in the pearl oyster Pinctada margaritifera var. cumingii (Linnaeus 1758) using a comparative whole genome pool-sequencing approach – <i>Evolutionary Applications, https://doi.org/10.1111/eva.13464</i>'

